ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh

```diff
@@ -22,8 +23,46 @@ set -x
 KCLI_PARAM=""
 if [ ! -z $OO_CHANNEL ] ; then
     KCLI_PARAM="-P openshift_image=registry.ci.openshift.org/ocp/release:$OO_CHANNEL"
+elif [ ! -z $JOB_NAME ]; then
+    tmpvar="${JOB_NAME/*nightly-/}"
+    ocp_ver="${tmpvar/-e2e-telco5g/}"
+    KCLI_PARAM="-P openshift_image=registry.ci.openshift.org/ocp/release:$ocp_ver"
+fi
+
+cat << EOF > ~/get-cluster-name.yml
+---
+- name: Grab and run kcli to install openshift cluster
+  hosts: all
+  gather_facts: false
+  tasks:
+  - name: Discover cluster to run job
+    command: python3 ~/telco5g-lab-deployment/scripts/upstream_cluster.py --get-cluster
+    register: cluster
+    environment:
+      JOB_NAME: ${JOB_NAME:-'unknown'}
+
+  - name: Create a file with cluster name
+    shell: echo "{{ cluster.stdout }}" > $SHARED_DIR/cluster_name
+    delegate_to: localhost
+EOF
+
+ansible-playbook -i ~/inventory ~/get-cluster-name.yml -vv
+CLUSTER_NAME=$(cat $SHARED_DIR/cluster_name)
+
+if [[ "$CLUSTER_NAME" == "no_cluster" || "$CLUSTER_NAME" == "" ]]; then
+    echo "No cluster for job, exiting! CLUSTER_NAME=${CLUSTER_NAME}"
+    exit 1
+fi
+
+if [[ "$CLUSTER_NAME" == "cnfdc5" ]]; then
+    TEST_CLUSTER_API_IP=10.19.16.74
```

Comment:
I dont like the internal IPs here
cant we use the kcli param file for that? 
ie - have here only env name?

Reply:
kcli param file is on HV, for getting to HV you need its IP.